filesystems/drs/src/test/scala/cromwell/filesystems/drs/DrsPathBuilderSpec.scala

```diff
@@ -277,27 +293,79 @@ class DrsPathBuilderSpec extends TestKitSuite with AnyFlatSpecLike with Matchers
       name = "",
       getFileName = null,
       getNameCount = 1,
-      isAbsolute = false),
+      isAbsolute = false,
+    ),
 
     GoodPath(
       description = "an absolute path without a host",
       path = s"drs://blah",
       normalize = false,
-      pathAsString = s"drs://blah/",
-      pathWithoutScheme = s"blah/",
+      pathAsString = s"drs://blah",
+      pathWithoutScheme = s"blah",
       parent = null,
       getParent = null,
-      root = s"drs://blah/",
+      root = s"drs://blah",
       name = "",
       getFileName = null,
       getNameCount = 1,
-      isAbsolute = false)
+      isAbsolute = false,
+    ),
+
+    // No spec says this is illegal... so pass it to Martha's various GCFs JIC
```

Comment:
I would assume that this literally cannot end well if we try it in production though...?

Reply:
After last week, what is well? 😉 

It shouldn't break the system, and the client should receive a very swift response of "Invalid 'url'" or something to that effect from `martha_v3`. The other GCFs I have no clue about, but Cromwell doesn't talk to them.